config/webpack/webpack.config.js

```diff
@@ -136,7 +137,10 @@ const buildWebpackConfigs = builds.map(
       .value();
 
     const internalJs = [paths.src, ...paths.compilePackages];
-    const entry = [paths.clientEntry];
+    const resolvedPolyfills = polyfills.map(polyfill => {
+      return require.resolve(polyfill, { paths: [process.cwd()] });
+    });
+    const entry = [...resolvedPolyfills, paths.clientEntry];
```

Comment:
Is this change related to the webpack 4 upgrade? Seems unrelated—do we know why this was changed?

Reply:
I cannot remember this change, @mbfisher . But if I did, it should be to clean it up as what @markdalgleish mentioned.

Reply:
👍

Reply:
🤔 how come `entry = [...devServerEntries, ...resolvedPolyfills, paths.clientEntry]`?
should be `entry = [ ...resolvedPolyfills, ...devServerEntries, paths.clientEntry]` 
☝️  @markdalgleish

Reply:
@hscgavin good point

Reply:
the order here is important 👇 

>  it's important for polyfills that modify the global environment to be loaded before all other code. To address this, the polyfills option allows you to provide an array of modules to import before any other code is executed.